gwtlib/worktrees.py

```
@@ -152,7 +170,106 @@ def switch_branch(branch_name, git_dir, create=False, force_create=False, guess=
+def _is_worktree_locked(worktree_path: str, git_dir: str) -> bool:
+    """Check if worktree is locked."""
+    try:
+        from gwtlib.parsing import parse_worktree_porcelain
+
+        entries = parse_worktree_porcelain(git_dir, include_main=False)
+        if entries:
+            for entry in entries:
+                if entry.get("path") == worktree_path:
+                    return entry.get("locked", False)
+    except Exception:
+        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)